grunt/tasks/build.js

```diff
@@ -28,7 +28,7 @@ module.exports = function(grunt) {
 
 	// Full distribution task
 	grunt.registerTask('dist', 'Build "dist." Contributors: do not commit "dist."',
-			['clean:dist', 'distcss', 'copy:fonts', 'copy:templates', 'distjs', 'commonjs', 'distzip']);
+			['clean:dist', 'distcss', 'copy:fonts', 'copy:templates', 'distjs', 'distzip']);
```

Comment:
Is the commonjs bundle used anywhere else in the dev stuff?  Meaning... is it _ok_ that we're removing it from dist completely and only running it during test?